fluent-package/msi/assets/fluent-package-post-migration.bat

```diff
@@ -0,0 +1,46 @@
+@echo on
+title Fluent-package post migration script
+
+if exist "%~dp0..\..\td-agent" (
+  echo "Migrate c:\opt\td-agent\etc\td-agent.conf"
+  if exist "%~dp0..\..\td-agent\etc\td-agent\td-agent.conf" (
+     move /Y "%~dp0..\..\td-agent\etc\td-agent\td-agent.conf" "%~dp0..\etc\fluent\td-agent.conf"
+  )
+  echo "Create c:\opt\fluent\var\log\fluent directory"
+  if not exist "%~dp0..\var\log\fluent" (
+     mkdir "%~dp0..\var\log\fluent"
+  )
+  echo "Migrate c:\opt\td-agent\var\log\td-agent\* files"
+  for %%f in (%~dp0..\..\td-agent\var\log\td-agent\*) do (
+     move /Y %%f "%~dp0..\var\log\fluent\"
+  )
+  echo "Migrate c:\opt\td-agent\var\log\td-agent\* directory (e.g buffer)"
+  for /d %%d in (%~dp0..\..\td-agent\var\log\td-agent\*) do (
+     move /Y %%d "%~dp0..\var\log\fluent\"

```

Comment:
> Move log files to c:\opt\fluent\var\log\fluent.

I think this is the logic.
This logic certainly moves `var\log\` files, but they are actually the buffer files only.
Currently, log files are directly under `C:\opt\td-agent\`, and they are lost!
We should consider the log files migration.

* `C:\opt\td-agent\td-agent-supervisor-0.log`
* `C:\opt\td-agent\td-agent-0.log`

Reply:
Thanks. I'll change it.